src/Hl7.Fhir.ElementModel/ScopedNode.cs

```diff
@@ -25,24 +25,28 @@ private class Cache
             public string? Id;
             public IEnumerable<ScopedNode>? ContainedResources;
             public IEnumerable<BundledResource>? BundledResources;
+
+            public string? InstanceUri;
         }
 
         private readonly Cache _cache = new();
 
         public readonly ITypedElement Current;
 
-        public ScopedNode(ITypedElement wrapped)
+        public ScopedNode(ITypedElement wrapped, string? instanceUri = null)
```

Comment:
Is this not breaking the interface? I see there is a nullable second parameter, but the definition of this constructor has changed. Should it not be better to add a second constructor:
```c#
public ScopedNode(ITypedElement wrapped, string instanceUri )
{
   // etc...
}

Reply:
That's equivalent - the second parameter is not only nullable, but also optional (`= null`).

Reply:
BTW: It is not *binary* compatible, but that was never a goal...